x-pack/plugins/ml/public/application/data_frame_analytics/common/get_index_data.ts

```diff
@@ -52,22 +53,19 @@ export const getIndexData = async (
         index: jobConfig.dest.index,
         body: {
           fields: ['*'],
-          _source: jobConfig.dest.results_field,
+          _source: [],
           query: searchQuery,
           from: pageIndex * pageSize,
           size: pageSize,
           ...(Object.keys(sort).length > 0 ? { sort } : {}),
         },
       });
 
-      setRowCount(resp.hits.total.value);
-      const docs = resp.hits.hits.map((d) => ({
-        ...getProcessedFields(d.fields),
-        [jobConfig.dest.results_field]: d._source[jobConfig.dest.results_field],
-      }));
-
-      setTableItems(docs);
-      setStatus(INDEX_STATUS.LOADED);
+      if (!options.didCancel) {
```

Comment:
For regression and classification jobs, the results grid is failing to display for me:

![image](https://user-images.githubusercontent.com/7405507/96577687-e1bf3e00-12cb-11eb-9731-bca132022a7a.png)